profile/2017-03-09-profile.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
```

Comment:
Do we need to mandate specifying empty types: {}

Reply:
There is no schema for the profile AFAIK. What docs did you use to create this file?

Reply:
we dont have schema yet. This is just a result of internal discussion with SDK team

Reply:
Keeping empty types{} could be helpful for easy parsing , Im keeping it for now. we can discuss this in our meeting